opensquirrel/circuit.py

```diff
@@ -110,17 +109,15 @@ def test_get_circuit_matrix(self) -> np.ndarray:
         * this matrix has 4**n elements, where n is the number of qubits
         * therefore this function is only here for testing purposes on small number of qubits
         * result is stored as a numpy array of complex numbers
-        """
 
+        """
         return circuit_matrix_calculator.get_circuit_matrix(self.squirrel_ir)
 
     def __repr__(self) -> str:
         """Write the circuit to a cQasm3 string."""
-
         return writer.squirrel_ir_to_string(self.squirrel_ir)
 
-    def export(self, format: ExportFormat):
-        if format == ExportFormat.QUANTIFY_SCHEDULER:
+    def export(self, fmt: ExportFormat = None) -> None:
```

Comment:
Isn't `format` a better name?

Reply:
`format` is used for the Python built-in string formatting method, so not smart to use it as a parameter name.

Reply:
Ah OK! Fair enough.